indica/available_quantities.py

```diff
@@ -208,4 +208,37 @@
     },
     # "get_astra": {  # TODO: implement
     # },
+
+    "get_transp": {  
+        #Equilibrium part
+        "t": ("t", ["t"]),
+        "psin": ("psin", []),
+        "index": ("index", []),
+        "R": ("R", []),
+        "z": ("z", []),
+        "rgeo": ("R_geo", ["t"]),
+        "rmag": ("R_mag", ["t"]),
+        "zmag": ("z_mag", ["t"]),
+        "psi_axis": ("poloidal_flux_axis", ["t"]),
+        "psi_boundary": ("poloidal_flux_boundary", ["t"]),
+        "ipla": ("plasma_current", ["t"]),
+        "wp": ("equilibrium_stored_energy", ["t"]),
+        "rbnd": ("R_boundary", ["t", "index"]),
+        "zbnd": ("z_boundary", ["t", "index"]),
+        "f": ("equilibrium_f", ["t", "psin"]),
+        "ftor": ("toroidal_flux", ["t", "psin"]),
+        "rmji": ("R_hfs", ["t", "psin"]),
+        "rmjo": ("R_lfs", ["t", "psin"]),
+        "volume": ("volume_jacobian", ["t", "psin"]),
+        "area": ("area_jacobian", ["t", "psin"]),
+        "psi": ("poloidal_flux", ["t", "z", "R"]),
+
```

Comment:
You're missing RHOTOR which is the native coordinate of all quantities in PROFILES.RHOTOR - remember that while PSIN is the coordinate of all 1D profiles in an equilibrium code (e.g. EFIT), for TRANSP and ASTRA it's RHOTOR.

This also means that all the quantities that here have PSIN as spatial coordinate, should instead have RHOTOR.